### ISSUE TITLE
hyperparameter λ

### ISSUE THREAD

**Issue by healthysong** [lmb-freiburg/Contrastive-Future-Trajectory-Prediction#1]
How to determine the hyperparameter λ on different data sets

**Comment by os1a** [comment]
Hi @songyongkang,

Thanks for your interest in our work.
As for the hyper-parameter lambda, we include in the supplementary material an ablation study on the ETH-UCY dataset. In case of another dataset, you may use cross-validation to determine the best value.
Generally, you have to select it such that it is not very large and overtakes the supervised loss term and not too small such that it has no effect.

Hope this answers your question,

**Comment by healthysong** [comment]
Thank you for your reply, and looking forward to the follow-up update of this code repository.
